.travis.yml

```diff
@@ -1,37 +1,41 @@
 language: python
 
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7-dev"
-
-env:
-  - COV_CORE_SOURCE=secretsanta
-  - COV_CORE_CONFIG=.coveragerc
+# https://blog.travis-ci.com/2017-05-11-introducing-build-stages
+matrix:
+  include:
+    - python: "3.5"
+      env:
+        - COV_CORE_SOURCE=secretsanta
+        - COV_CORE_CONFIG=.coveragerc
+    - python: "3.6"
+      env:
+        - COV_CORE_SOURCE=secretsanta
+        - COV_CORE_CONFIG=.coveragerc
+    - python: "3.7"
+      env:
+        - COV_CORE_SOURCE=secretsanta
+        - COV_CORE_CONFIG=.coveragerc
+    - python: "3.8-dev"
+      env:
+        - COV_CORE_SOURCE=secretsanta
+        - COV_CORE_CONFIG=.coveragerc
```

Comment:
If the `env` block is defined outside of the `matrix` and only once, Travis will execute several builds per Python version, once with each environment variable set individually.